services/deep-crawl-service/tests/test_crawler.py

```
@@ -0,0 +1,150 @@
+class TestCrawlStrategies:
+    """Test different crawl strategies."""
+
+    @pytest.fixture
+    async def coordinator(self):
+        """Create coordinator instance."""
+        coord = DeepCrawlCoordinator()
+        yield coord
+        await coord.close()
+
+    @pytest.mark.asyncio
+    async def test_crawl_bfs_basic(self, coordinator, monkeypatch):
```

Ruff (0.14.0)

Unused method argument: `monkeypatch`

(ARG002)